src/style.css

```diff
@@ -0,0 +1,179 @@
+@font-face {
+  font-family: "Roboto";
+  src: url(Roboto/Roboto-Regular.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: "Roboto";
+  src: url(Roboto/Roboto-Medium.ttf);
+  font-style: normal;
+  font-weight: 500;
+}
+
+@font-face {
+  font-family: "Roboto";
+  src: url(Roboto/Roboto-Bold.ttf);
+  font-style: normal;
+  font-weight: 700;
+}
+
+:root {
+  --color-blue: #00acdc;
+  --color-white: #fff;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+h1,
+h2,
+h3 {
+  margin: 0;
+  padding: 0;
+}
+
+.body {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  margin: 0 16px;
+}
+
+.card__img {
+  margin-top: 32px;
+}
+
+.card__img-logo {
+  width: 160px;
+  height: 134px;
+}
+
+.card__name {
+  margin-top: 40px;
+  padding: 0 16px;
+  width: 100%;
+  
+  font-size: 12px;
+  line-height: 18px;
+
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  color: #060b35;
+}
+
+.card__product-code {
+  margin-top: 4px;
+  width: 166px;
+  
+  font-size: 10px;
+  line-height: 14px;
+
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  color: #616070;
+}
+
+.card__reviews {
+  margin-top: 16px;
+
+  width: 166px;
+  height: 16px;
+
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__reviews-stars {
+  display: flex;
+  align-items: center;
+}
+
+.card__reviews-quantity {
+  font-size: 10px;
+  line-height: 14px;
+  
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  color: #060b35;
+
+  display: flex;
+  align-items: center;
+  
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+
+  
+}
+
+.stars .stars__star:nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+  margin-right: 4px;
+}
+
```

Comment:
Stars should work same as you do [here](https://github.com/AndriiTorhashov/layout_stars)